README.md

```diff
@@ -1,63 +1,63 @@
-# BoldBI Embedding Blazor server sample
+# Bold BI Embedded Sample in Blazor Server
 
-This sample demonstrates how to embed the desired dashboard that is available in your Bold BI server.
+This project was created using ASP.NET Core 6.0. The purpose of this application is to demonstrate how to render the dashboard available on your Bold BI server.
 
-This section guides you in using the Bold BI dashboard in your Blazor server sample application.
+## Dashboard view
 
- * [Requirements to run the demo](#requirements-to-run-the-demo)
- * [Using the Blazor server sample](#using-the-Blazor-server-sample)
- * [Online Demos](#online-demos)
- * [Documentation](#documentation)
+![Dashboard View](https://github.com/boldbi/aspnet-core-sample/assets/91586758/4af68f49-ffc0-400a-a323-55a3f3600a1d)
 
- ## Requirements to run the demo
+ ## Requirements/Prerequisites
 
-The samples require the following requirements to run.
+ * [.NET Core 6.0](https://dotnet.microsoft.com/download/dotnet-core)
 
- * [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/)
- * [.NET Core 6.0 or later](https://dotnet.microsoft.com/en-us/download/dotnet-core)
+ #### Help link
 
- ## Using the Blazor server sample
+ * https://help.boldbi.com/embedded-bi/faq/where-can-i-find-the-product-version/
+
+ #### Supported browsers
+  
+  * Google Chrome, Microsoft Edge, Mozilla Firefox, and Safari.
+
+ ## Configuration
+
+  * Please ensure that you have enabled embed authentication on the `embed settings` page. If it is not currently enabled, please refer to the following image or detailed [instructions](https://help.boldbi.com/site-administration/embed-settings/#get-embed-secret-code) to enable it.
```

Comment:
* Please ensure you have enabled embed authentication on the `embed settings` page. If it is not currently enabled, please refer to the following image or detailed [instructions](https://help.boldbi.com/site-administration/embed-settings/#get-embed-secret-code) to enable it.

Reply:
Content updated